### ISSUE TITLE
error in parsing while " is firs char in row text and " is a part of column text

### ISSUE THREAD

**Issue by kashiash** [phatcher/CsvReader#56]
this row is parsed correctly:

6224613;SINUS POLSKA, MIEDZYRZECZ;6224613;10020;TOTAL Deutschland GmbH Berlin;2018-05-07;20:28:01;DEU;3292;38179;TOTAL,Schwülper;TOTAL;19;FMI 35435;178109;;;FULLSERV;8183;70600092461381836;01140;high performance diesel fuel;74.33;EUR;1.1084;1.319;82.39;98.04;2186686583;0381700

this is not parsed correctly:
6224613;"SINUS POLSKA", MIEDZYRZECZ;6224613;10020;TOTAL Deutschland GmbH Berlin;2018-05-07;20:28:01;DEU;3292;38179;TOTAL,Schwülper;TOTAL;19;FMI 35435;178109;;;FULLSERV;8183;70600092461381836;01140;high performance diesel fuel;74.33;EUR;1.1084;1.319;82.39;98.04;2186686583;0381700

but this is parsed corectly too:

6224613;SINUS "POLSKA", MIEDZYRZECZ;6224613;10020;TOTAL Deutschland GmbH Berlin;2018-05-07;20:28:01;DEU;3292;38179;TOTAL,Schwülper;TOTAL;19;FMI 35435;178109;;;FULLSERV;8183;70600092461381836;01140;high performance diesel fuel;74.33;EUR;1.1084;1.319;82.39;98.04;2186686583;0381700

this means that if column contains " at first character then i get error ... 
the csv is created by external source and i cant change this

**Comment by phatcher** [comment]
Looks like it might be a bug - see if this case is covered off by a unit test

**Comment by jakubmaguza** [comment]
I don't think this is an issue. `CsvReaderSampleData.SampleData1` contains data with quotation mark at first  character and it's being used in unit tests.

**Comment by jakubmaguza** [comment]
https://github.com/phatcher/CsvReader/pull/61

I checked this issue, and it's as follow:
If you have quote at the beginning of your field, then parser expects that next quote will be field closing quote. The only case for test to pass is to escape quote inside and add quote at the end of field like this:
"good,Also\"good\",\"wrong\"\"Field\",goodToo".
In that case wrong Field will be parsed properly.
I will try to work on it if I may.
